bin/pycbc_multi_inspiral

```diff
@@ -86,89 +110,254 @@ psd.insert_psd_option_group_multi_ifo(parser)
 scheme.insert_processing_option_group(parser)
 fft.insert_fft_option_group(parser)
 
+#Adding these to see if it lets me use the gating options
+from pycbc.vetoes.sgchisq import SingleDetSGChisq
+pycbc.opt.insert_optimization_option_group(parser)
+pycbc.weave.insert_weave_option_group(parser)
+pycbc.inject.insert_injfilterrejector_option_group(parser)
+SingleDetSGChisq.insert_option_group(parser)
 opt = parser.parse_args()
 
+generate_histograms = False
+generate_snr_plots = False
+generate_chi2_plots = False
```

Comment:
These flags, and the large blocks of plotting at the bottom, look like debugging. Please remove that.